scripts/install

```diff
@@ -32,6 +32,9 @@ ynh_script_progression --message="Setting up source files..." --weight=5
 ynh_setup_source --dest_dir="$install_dir"
 ynh_setup_source --dest_dir="$install_dir/addon" --source_id="addons"
 
+mkdir -p "$install_dir/view/smarty3"
+
+chmod -R 775 "$install_dir/view/smarty3"
```

Comment:
I'm still confused by this, is it meant to be the only folder that the `$app` user has write access to in the whole install folder?

Reply:
no `chmod -R 770 "$install_dir"` permission is set automatically at install.